test/models/dspace_metadata_test.rb

```diff
@@ -56,138 +70,251 @@ def dss_friendly_thesis(thesis)
     # removed once we update the Thesis validations to reflect publication requirements.
     t2.advisors = [advisors(:first)]
     t2.save
-    dc = DspaceMetadata.new(t2).instance_variable_get(:@dc)
-    assert_equal %w[0002 0001], dc['dc.identifier.orcid']
+    serialized = DspaceMetadata.new(t2).serialize_dss_metadata
+    unserialized = JSON.parse(serialized)
+
+    assert unserialized['metadata'].include?({ 'key' => 'dc.identifier.orcid', 'value' => '0001' })
+    assert unserialized['metadata'].include?({ 'key' => 'dc.identifier.orcid', 'value' => '0002' })
 
     # Multiple authors and only one ORCID
-    t2.users.second.orcid = nil
-    t2.users.second.save
-    dc = DspaceMetadata.new(t2).instance_variable_get(:@dc)
-    assert_equal ['0002'], dc['dc.identifier.orcid']
+    u = User.find_by_orcid('0002')
+    u.orcid = nil
+    u.save
+    t2.reload
+    serialized = DspaceMetadata.new(t2).serialize_dss_metadata
+    unserialized = JSON.parse(serialized)
+
+    assert unserialized['metadata'].include?({ 'key' => 'dc.identifier.orcid', 'value' => '0001' })
+    refute unserialized['metadata'].include?({ 'key' => 'dc.identifier.orcid', 'value' => '0002' })
 
     # One author and no ORCID
```

Comment:
I see what the changes to the test do, and don't object - but this is no longer a test of one author and no ORCIDs - there are two authors in the t2 fixture at this stage, neither of which have an ORCID. The tests seem to confirm that zero ORCIDs are present, which I think is the correct test, but the comment probably needs to be updated.

Reply:
I suspect I copy/pastad the wrong comment. I'll take a closer look. Thanks for catching that.